roles/disaster_recovery/files/ovirt-dr

```diff
@@ -123,7 +119,7 @@ def _get_log_conf(conf_file, log_file, log_level):
         log_level = settings.get(log_section, log_level_conf)
     else:
         log_level = "DEBUG"
-    return log_file, log_level
+    return conf_file, log_file, log_level
```

Comment:
Why are you mixing this enhancement with a bug fix? It is best to keep it in a separate PR so we can merge the fix quickly.

The issue is not clear, maybe you need to show it in the commit message - how the interaction looks before and after the change.

Reply:
OK, I will separate it to 2 PRs after lunch, thanks.
The previous behavior is like that
```
[root@storage-ge-5 files]# ./ovirt-dr generate
Conf file 'dr.conf' does not exist. Please provide the configuration file location: dr1.conf           
Conf file 'dr.conf' does not exist. Please provide the configuration file location: dr1.conf
Log file: '/tmp/ovirt-dr-1653552400881.log'
[Generate Mapping File] Password is not initialized. Please provide the password for username admin@internal: 123456
...
```

Reply:
IMHO, I think it would be better to refactor the part that verifies that the config file exists and ask for the location if it doesn't out of `_get_log_conf`, which would always receive an existing file

Reply:
@aesteve-rh That's a good idea, will need to be a separate PR though :)
Also looks like `README.md` and `dr.conf` skeleton might also require a small update in regards to log filename and level for consistency between the 4 DR flows.